packages/react-cards/src/components/Card/CardItem/CardItem.types.ts

```diff
@@ -0,0 +1,66 @@
+import { IComponent, IComponentStyles, ISlotProp, IStyleableComponentProps } from '@uifabric/foundation';
+import { IBaseProps } from '@uifabric/utilities';
+import { IStackItemProps, IStackItemSlots, IStackItemTokens } from 'office-ui-fabric-react/lib/Stack';
+
+/**
+ * {@docCategory CardItem}
+ */
+export type ICardItemComponent = IComponent<ICardItemProps, ICardItemTokens, ICardItemStyles, ICardItemViewProps>;
+
+// These types are redundant with ICardItemComponent but are needed until TS function return widening issue is resolved:
+// https://github.com/Microsoft/TypeScript/issues/241
+// For now, these helper types can be used to provide return type safety when specifying tokens and styles functions.
+
+/**
+ * {@docCategory CardItem}
+ */
+export type ICardItemTokenReturnType = ReturnType<Extract<ICardItemComponent['tokens'], Function>>;
+
+/**
+ * {@docCategory CardItem}
+ */
+export type ICardItemStylesReturnType = ReturnType<Extract<ICardItemComponent['styles'], Function>>;
+
+/**
+ * {@docCategory CardItem}
+ */
+export type ICardItemSlot = ISlotProp<ICardItemProps>;
+
+/**
+ * {@docCategory CardItem}
+ */
+export interface ICardItemSlots extends IStackItemSlots {}
+
+/**
+ * {@docCategory CardItem}
+ */
+export interface ICardItem {}
+
+/**
+ * {@docCategory CardItem}
+ */
+export interface ICardItemProps
+  extends ICardItemSlots,
+    Pick<IStackItemProps, 'grow' | 'shrink' | 'disableShrink' | 'align' | 'verticalFill' | 'order'>,
+    IStyleableComponentProps<ICardItemProps, ICardItemTokens, ICardItemStyles>,
+    IBaseProps<ICardItem> {
+  /**
+   * Determines if the CardItem should disregard the children margin specified by Card.
+   */
+  fill?: boolean;
```

Comment:
Is fill a concept StackItem would never have?

Reply:
Well, `fill` is specifically for bleeding into the edge of the `Card`, which I think is a concept inherently for `CardItem`/`CardSection` and not `StackItem`, since `Card` has a say in it's children's margin while `Stack` does not.